### PR TITLE
Remove references to MLflow UI in getting started guide and replace i…

### PR DESCRIPTION
**Pull request by BenWilson2**
<details><summary>&#x1F6E0 DevTools &#x1F6E0</summary>
<p>

[Open in GitHub Codespaces](https://codespaces.new/BenWilson2/mlflow/pull/10231?quickstart=1)

#### Install mlflow from this PR

```
pip install git+https://github.com/mlflow/mlflow.git@refs/pull/10231/merge
```

#### Checkout with GitHub CLI

```
gh pr checkout 10231
```

</p>
</details>

…mage references

### Related Issues/PRs

 #xxx

### What changes are proposed in this pull request?

Remove the redundant references to MLflow UI in the getting started guide and replace images that reference it. 
Also, address the ToC rendering inconsistency in the notebook contents for the getting started guide

### How is this PR tested?

- [ ] Existing unit/integration tests
- [ ] New unit/integration tests
- [X] Manual tests

### Does this PR require documentation update?

- [ ] No. You can skip the rest of this section.
- [X] Yes. I've updated:
  - [X] Examples
  - [ ] API references
  - [ ] Instructions

#### Is this a user-facing change?

- [X] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

#### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [X] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/gateway`: AI Gateway service, Gateway client APIs, third-party Gateway integrations
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

#### How should the PR be classified in the release notes? Choose one:

- [X] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: docs/source/getting-started/logging-first-model/step1-tracking-server.rst [hunk 1]
```diff
@@ -38,26 +38,12 @@ running during the tutorial, as closing it will shut down the server.
...
-        mlflow ui --host 127.0.0.1 --port 8090
-
-.. figure:: ../../_static/images/tutorials/introductory/logging-first-model/start-servers.gif
-   :width: 1024px
-   :align: center
-   :alt: Starting the MLflow Tracking and UI servers
-
-   Starting the MLflow Tracking Server and the MLflow UI
-
+.. figure:: ../../_static/images/tutorials/introductory/logging-first-model/start-mlflow-server.png
```
Comment (BenWilson2) [inline docs/source/getting-started/logging-first-model/step1-tracking-server.rst:41]: got rid of the silly .gif here. Static image is easier / better.

File: docs/source/getting-started/logging-first-model/step3-create-experiment.rst [hunk 2]
```diff
@@ -5,21 +5,22 @@ In the previous section, we became familiar with the MLflow Client and its ``sea
...
 
 .. code-block:: text
 
-        http://127.0.0.1:8090
+        http://127.0.0.1:8080
 
 Once navigating to the url for the MLflow UI, you will see the default experiment with no run data.
 
-.. figure:: ../../_static/images/tutorials/introductory/logging-first-model/default-ui.gif
+.. figure:: ../../_static/images/tutorials/introductory/logging-first-model/default-ui.png
```
Comment (BenWilson2) [inline docs/source/getting-started/logging-first-model/step3-create-experiment.rst:23]: switch to static file

File: docs/source/llms/llm-evaluate/notebooks/index.rst [hunk 3]
```diff
@@ -12,11 +12,6 @@ toxicity as well as LLM-judged intelligent metrics such as answer_relevance.
     :hidden:
 
     question-answering-evaluation.ipynb
-
-.. toctree::
```
Comment (BenWilson2) [inline docs/source/llms/llm-evaluate/notebooks/index.rst:16]: this was redundant

File: docs/source/_static/images/tutorials/introductory/logging-first-model/start-mlflow-server.png
Comment (harupy) [inline docs/source/_static/images/tutorials/introductory/logging-first-model/start-mlflow-server.png:1]: do we need this image? To me, it's redundant.

File: docs/source/getting-started/logging-first-model/step3-create-experiment.rst [hunk 4]
```diff
@@ -5,21 +5,22 @@ In the previous section, we became familiar with the MLflow Client and its ``sea
...
-host as ``127.0.0.1`` and the port as ``8090``.  Let's go to the UI and see what the Default Experiment looks like.
+In the first section of this tutorial, we started the MLflow Tracking Server from a command prompt, specifying the
+host as ``127.0.0.1`` and the port as ``8080``.  Let's go to the UI and see what the Default Experiment looks like.
 
 Viewing the MLflow UI
 ---------------------
 
-In order to navigate to the MLflow UI server, we simply have to use a web browser and enter:
+In order to see the MLflow UI, we simply have to use a web browser to connect to the MLflow Tracking Server. 
+Open your browser and enter:
```
Comment (harupy) [inline docs/source/getting-started/logging-first-model/step3-create-experiment.rst:15]:
```suggestion
Open http://127.0.0.1:8080.
```

**Review by harupy — APPROVED** [review]

**Pull request merged**